package.json

```diff
@@ -40,10 +40,12 @@
     "homepage": "https://github.com/textkernel/oneui#readme",
     "dependencies": {
         "@babel/polyfill": "7.0.0",
-        "css-vars-ponyfill": "1.16.4",
-        "prop-types": "15.6.2",
-        "react": "16.8.6",
-        "react-dom": "16.8.6"
+        "css-vars-ponyfill": "1.16.4"
+    },
+    "peerDependencies": {
+        "prop-types": ">=15.6.2",
```

Comment:
I'm not sure this is right, this would mean OneUI is also compatible with new major versions (with breaking changes) of these packages?

Shouldn't it be like `^15.6.2` instead? (Same for dependencies below)